backends/cadence/aot/functions_hifi.yaml

```diff
@@ -77,7 +77,17 @@
     - arg_meta: null
       kernel_name: torch::executor::max_pool2d_with_indices_out
 
+- op: maximum.out
+  kernels:
+    - arg_meta: null
+      kernel_name: impl::HiFi::maximum_out
```

Comment:
Please change all kernel_names here to match cadence::impl::HiFi::native::OP_NAME. For example: cadence::impl::HiFi::native::add_out